src/config_db/SCHEMA.md

```diff
@@ -33,7 +33,7 @@ Contents:
         "tag": "0.1.0",
     }
     "parameters": { ... }
-    "scanParameters:
+    "scanParameters":
```

Comment:
We are going to need to change the example at https://confluence.skatelescope.org/pages/viewpage.action?pageId=74716479 still for the change between underscore and camelCase notation for parameters.